### PR TITLE
Update QR Tab's Styling for input boxes.

### PR DESCRIPTION
**Pull request by KashSingh1**
Everything nearly works, but there is one problem.

Currently, it's not possible to submit from either the desktop view and the mobile. You have to pick one and disable the other. The reason for this, I believe, is that`ref={inputEl}` appears in two different places. Because of this, only the last place it appears will have working code, the previous occurrence (desktop view) won't work.

I think it possible to fix this just by duplicating the function and split `inputEl` into two different variables, and have one variable for each function. Other than that, the branch is ready to merge.

### CODE REVIEW

File: src/components/project-tabs/QRTab.jsx [hunk 1]
```diff
@@ -275,7 +275,7 @@ const QRTab = (props) => {
           >
             <InputGroup className="input-group-prepend">
               <FormControl
-                ref={inputEl}
+                // ref={inputEl} Can't have inputEl in both places
```
Comment (KashSingh1) [inline src/components/project-tabs/QRTab.jsx:278]: The second occurrence of `InputEl`.  I disabled it so the desktop version submit would work properly.

File: src/components/project-tabs/QRTab.jsx [hunk 2]
```diff
@@ -73,6 +73,7 @@ const QRTab = (props) => {
   // Create QR code link
   const qrCodeLink = `exp://exp.host/@imaginethis/${state.projectID}`;
 
+  lastConversion = "7865"
```
Comment (KashSingh1) [inline src/components/project-tabs/QRTab.jsx:107]: Delete when ready to merge. Only for testing purposes.

File: src/components/project-tabs/QRTab.jsx [hunk 3]
```diff
@@ -164,6 +146,24 @@ const QRTab = (props) => {
                 <li>A notification will appear saying to open the build in Expo. Click on this.</li>
                 <li>The expo app should then open and the prototype should begin to run on your device.</li>
               </ol>
+              <Form
+                onSubmit={sendEmail}
+                className="input-group pc-input-bar-group"
+              >
+                <InputGroup className="input-group-append">
+                  <FormControl
+                    ref={inputEl}
```
Comment (KashSingh1) [inline src/components/project-tabs/QRTab.jsx:200]: The first instance of `inputEl`

File: src/components/project-tabs/QRTab.jsx (hunk 1 above)
Comment (bbaayezi) [reply src/components/project-tabs/QRTab.jsx:278]: I can see the input field codes are actually duplicating, it is possible to remove any duplicate code and control the content to display on different devices using an internal state?

**Comment by bbaayezi** [comment]
Fix applied in another PR #43

**Comment by KashSingh1** [comment]
No longed needed.

**Pull request closed without merging**